Classes/Domain/Repository/ExtensionRepository.php

```diff
@@ -165,33 +166,29 @@ public function findByHasOpenOffice($allowedInstallTypes = 'G,L')
     public function findExtensionsBySearchTerm(array $extensionKeys, $searchTerm, $limit)
     {
         $extensionTable = 'tx_extensionmanager_domain_model_extension';
-        $extensions = $this->getDatabaseConnection()->exec_SELECTgetRows(
-            'DISTINCT extension_key, title',
-            $extensionTable,
-            $this->getSafeInClause('extension_key', array_map(function ($e) {
-                return "'" . $e . "'";
-            }, $extensionKeys)) .
-            ' AND ' . $this->getDatabaseConnection()->searchQuery(
-                array($searchTerm),
-                array('extension_key', 'title', 'description'),
-                $extensionTable
-            ),
-            '',
-            'extension_key, last_updated',
-            $limit,
-            'extension_key'
-        );
+
+        $qBuilder = GeneralUtility::makeInstance(QueryBuilder::class);
```

Comment:
`$queryBuilder`, not `$qBuilder`, just like everywhere in Core, there is no sense abbreviating it